internal/cloudapi/server.go

```diff
@@ -77,6 +77,20 @@ func (server *Server) Compose(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var bp = blueprint.Blueprint{}
+	err = bp.Initialize()
+	if err != nil {
+		http.Error(w, "Unable to initialize blueprint", http.StatusInternalServerError)
+		return
+	}
+	if request.Customizations != nil && request.Customizations.Packages != nil {
+		for _, p := range *request.Customizations.Packages {
+			bp.Packages = append(bp.Packages, blueprint.Package{
+				Name: p,
```

Comment:
@Gundersanne so, to make sure I understand this properly, when we create our image we will pass a list of package names as a customization. The list of names would look something like `['0ad', 'curl', 'openssh']` correct? In the future will there be a desire to support multiple package versions?

Reply:
Correct!
Of desires of the future I know not much. However, not supporting package versions was intentional from my side. The thinking was that we mostly don't have different versions of the same package in our repos (so selecting anything but the most recent one will just fail), and if we do, mixing and matching versions surely have not been tested well. So let's start off with only supporting the most recent versions, and if we want more granularity in the future we should discuss exactly what is wanted / what that means.